### ISSUE TITLE
About CAM loss

### ISSUE THREAD

**Issue by markduon** [znxlwm/UGATIT-pytorch#61]
Dear sir,

I am a little confused with 2 CAM loss functions (5) and (6) in your paper. I don't know why one function uses BCE for generator, one uses MSE for discriminator? Can someone explain it? Many thanks.

**Comment by zihaog0724** [comment]
same here.

**Comment by WangDingok** [comment]
cam loss in gen only for gen, not for both gen and dis ( adv loss ), so just a normal classification task for gen
